atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/StreamStoreRenderer.java

```diff
@@ -684,9 +686,26 @@ private void cellsCleanedUp() {
                     line("for (Cell cell : cells) {"); {
                         line("rows.add(", StreamIdxRow, ".BYTES_HYDRATOR.hydrateFromBytes(cell.getRowName()));");
                     } line("}");
-                    line("Multimap<", StreamIdxRow, ", ", StreamIdxColumnValue, "> rowsInDb = usersIndex.getRowsMultimap(rows);");
-                    line("Set<", StreamId, "> toDelete = Sets.newHashSetWithExpectedSize(rows.size() - rowsInDb.keySet().size());");
-                    line("for (", StreamIdxRow, " rowToDelete : Sets.difference(rows, rowsInDb.keySet())) {"); {
+
+                    line("BatchColumnRangeSelection oneColumn = BatchColumnRangeSelection.create(");
+                    line("        PtBytes.EMPTY_BYTE_ARRAY, PtBytes.EMPTY_BYTE_ARRAY, 1);");
+                    line("Map<", StreamIdxRow, ", BatchingVisitable<", StreamIdxColumnValue, ">> existentRows");
+                    line("        = usersIndex.getRowsColumnRange(rows, oneColumn);");
+
+                    line("Set<", StreamIdxRow, "> rowsInDb = Sets.newHashSetWithExpectedSize(cells.size());");
+
+                    line("for (Map.Entry<", StreamIdxRow, ", BatchingVisitable<", StreamIdxColumnValue, ">> rowVisitable");
+                    line("        : existentRows.entrySet())"); {
```

Comment:
nit: can we wrap the for loop content with brackets?

Reply:
also not sure if this is affecting spacing, but the next line isn't being indented like I'd expect it to

Reply:
Good call - the indentation is taken care of by the braces in the raw code-strings being there or not there. I left this out by accident (but the tests still passed, because the scope of the for loop happened to be 1 statement. New version should be OK (see generated code)